templates/js-crawlee-cheerio/src/main.js

```diff
@@ -7,19 +7,27 @@
 // For more information, see https://docs.apify.com/sdk/js
 import { Actor } from 'apify';
 // For more information, see https://crawlee.dev
-import { CheerioCrawler } from 'crawlee';
-import { router } from './routes.js';
+import { CheerioCrawler, Dataset } from 'crawlee';
 
 // Initialize the Apify SDK
 await Actor.init();
 
-const startUrls = ['https://apify.com'];
+const { startUrls, maxRequestsPerCrawl } = await Actor.getInput() || {};
 
 const proxyConfiguration = await Actor.createProxyConfiguration();
 
 const crawler = new CheerioCrawler({
     proxyConfiguration,
-    requestHandler: router,
+    maxRequestsPerCrawl,
+    requestHandler: async ({ enqueueLinks, request, $, log }) => {
```

Comment:
I'd personally rather see the ES6 shorthand syntax here, we use that in the crawlee docs.

```suggestion
    async requestHandler({ enqueueLinks, request, $, log }) {
```

less special characters => easier to read and write